### PR TITLE
Update prepare_data.sh

### PR DESCRIPTION
**Pull request by AnBucquet**
Related to #142.

### CODE REVIEW

File: prepare_data/README.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ In its current state, this pipeline uses [SCT development version](https://githu
 See [README](../README.md)
 ~~~
 source PATH_TO_YOUR_VENV/venv-ivadomed/bin/activate
+# install prepdata
+cd .. && python setup.py install && cd -
```
Comment (AnBucquet) [inline prepare_data/README.md]: It seems that [prepdata](https://github.com/neuropoly/ivado-medical-imaging/tree/master/prepare_data/prepdata) is not installed with ivadomed, which leads to the failure of `./run_process.sh parameters.sh final_qc.sh`.

File: prepare_data/README.md [hunk 2]
```diff
@@ -100,5 +102,5 @@ Copy final files to anat/, copy json sidecars, move segmentations to derivatives
 Once QC and manual correction is done, remove tmp/ folder:
 
 ~~~
-./run_process.sh delete_tmp_files.sh
+./run_process.sh parameters.sh delete_tmp_files.sh
```
Comment (jcohenadad) [inline prepare_data/README.md]: actually, we can now use `sct_run_batch`.

**Comment by AnBucquet** [comment]
Tested on **ub-balgrist02**, seems to work.
(cf. `/Duke/projects/csf_deepseg/SpineGe_debbug/result/sub-balgrist02`)

File: prepare_data/README.md (hunk 1 above)
Comment (jcohenadad) [inline prepare_data/README.md]: duplication with https://github.com/neuropoly/ivado-medical-imaging/blob/master/README.md#installing

Comment (jcohenadad) [reply prepare_data/README.md]: ah yes you're right... it's not the most intuitive way of organizing a software structure (i'm the one to blame), and the only dependent python software is actually `prepdata remove-slice` (called by `final_qc.sh`). So i guess until we find a better structure/organization (feel free to suggest), we should indeed keep that install documentation.

**Pull request merged**